src/ecies/ecies_salsa_sealed_box.rs

```diff
@@ -133,6 +133,33 @@ mod tests {
         assert_eq!(plaintext, &plaintext_[..]);
     }
 
+    #[test]
+    fn ecies_salsa_seal_box_ed() {
+        let mut rng = CsRng::from_entropy();
+        // Generate an ED25519 key pair
+        let ed25519_sk = Ed25519PrivateKey::new(&mut rng);
+        let ed25519_pk = Ed25519PublicKey::from(&ed25519_sk);
+
+        // generate an X25519 public key from the ED25519 public key
+        let x25519_pk = X25519PublicKey::from_ed25519_public_key(&ed25519_pk);
+        // encrypt
+        let plaintext = b"Hello World!";
+        let ciphertext = EciesSalsaSealBox::encrypt(&mut rng, &x25519_pk, plaintext, None).unwrap();
+        // check the size is the expected size
+        assert_eq!(
+            ciphertext.len(),
+            plaintext.len() + EciesSalsaSealBox::ENCRYPTION_OVERHEAD
+        );
+
+        // generate the corresponding X25519 private key from the ED25519 private key
+        let x25519_sk = X25519PrivateKey::from_ed25519_private_key(&ed25519_sk);
+
+        // decrypt
+        let plaintext_ = EciesSalsaSealBox::decrypt(&x25519_sk, &ciphertext, None).unwrap();
+        // assert
+        assert_eq!(plaintext, &plaintext_[..]);
```

Comment:
```suggestion
        let plaintext = EciesSalsaSealBox::decrypt(&x25519_sk, &ciphertext, None).unwrap();
        // assert
        assert_eq!(plaintext, &plaintext[..]);
```

Reply:
plaintext_ (with underscore) is the one recovered. plaintext (without underscore is the original)

Reply:
oops.. ok